sdk/keyvault/test-resources.json

```diff
@@ -3,14 +3,80 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "baseName": {
-      "type": "String"
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
     },
     "tenantId": {
-      "type": "String",
-      "defaultValue": "[subscription().tenantId]"
+      "type": "string",
+      "defaultValue": "[subscription().tenantId]",
+      "metadata": {
+        "description": "The tenant ID to which the application and resources belong."
+      }
     },
     "testApplicationOid": {
-      "type": "String"
+      "type": "string",
+      "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
+      "metadata": {
+        "description": "The client OID to grant access to test resources."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resource. By default, this is the same as the resource group."
+      }
+    },
+    "hsmLocation": {
+      "type": "string",
+      "defaultValue": "centralus",
```

Comment:
Given how long these can take to run, it might be a good idea to use a different region from .NET. Maybe "southceltralus" or "westus"? After the service GAs (or maybe it's already supported) it shouldn't matter as the 1 HSM/region/sub restriction should be lifted. Still, better safe than sorry for now.

Reply:
According to our shared document `centralus` is Java's designated region and `southcentralus` is .NET's, is there a different document I should be looking at?

Reply:
Hmm. Out of date then. But I also wrote that when our tests were only taking < 2 hours. For Managed HSM it's up to 4 because we have to run most admin tests sequentially. You can update the doc.